src/activities/rahulworld/Rahulworld.qml

```diff
@@ -0,0 +1,197 @@
+/* GCompris - rahulworld.qml
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   YOUR NAME <YOUR EMAIL> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import "../../core"
+import "rahulworld.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "#ABCDEF"
+        signal start
+        signal stop
+
+        Image {
+            id: bg
+            source: dataset.item.backgroundImage
+            sourceSize.width: 2000 * ApplicationInfo.ratio
+            sourceSize.height: 2000 * ApplicationInfo.ratio
+            width: 2000 * background.playRatio
+            height: width
+            anchors.centerIn: parent
+        }
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        GCText {
+            anchors.centerIn: parent
+            text: "Explore and Know about Enviroment"
```

Comment:
use qsTr() for translatable texts.
Also, the text should fit in the screen correctly, independantly of the screen resolution.